### PR TITLE
[Frequent Updates] Gradle 6.6.1 & Android Gradle Plugin 4.0.1 & com.jfrog.bintray.gradle 1.8.5

### PR DESCRIPTION
**Pull request by mosabab**
### Platforms affected
Android

### Motivation and Context

[To keep Android projects sync with latest Gradle versions and anything related to Gradle]

As Android Gradle plugin release notes https://developer.android.com/studio/releases/gradle-plugin#updating-gradle
Related to Gradle version and Android Gradle Plugin.

Here what the latest versions:

Updates:

- Gradle version to 6.6.1
- Android Gradle Plugin version to 4.0.1
- com.jfrog.bintray.gradle to 1.8.5

### Checklist

- [x] I've run the tests to see all new and existing tests pass
- [x] I added automated test coverage as appropriate for this change
- [x] Commit is prefixed with `(platform)` if this change only applies to one platform (e.g. `(android)`)
- [x] If this Pull Request resolves an issue, I linked to the issue in the text above (and used the correct [keyword to close issues using keywords](https://help.github.com/articles/closing-issues-using-keywords/))
- [x] I've updated the documentation if necessary

### CODE REVIEW

**Comment by mosabab** [comment]
@breautek 
[INVITED TO REVIEW]

File: bin/templates/project/build.gradle [hunk 1]
```diff
@@ -19,13 +19,13 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.3.50'
+    ext.kotlin_version = '1.4.10'
```
Comment (timbru31) [inline bin/templates/project/build.gradle:22]: Personally I don't mind updating the complete toolset in one PR, but one could argue that either it's listed in the PR description and title or updated in a separate PR.

Comment (mosabab) [reply bin/templates/project/build.gradle:22]: Your idea is correct, but this PR update any related to Gradle, I update the title to all updates affected and also I put the details in description filed.

Comment (breautek) [reply bin/templates/project/build.gradle:22]: Google does not follow traditional semver rules and minor releases can include breaking changes, therefore we cannot upgrade the minor version of Kotlin without also introducing a breaking change.

Comment (mosabab) [reply bin/templates/project/build.gradle]: I revert the kotlin version to 1.3.50.

So the other should keep updated to dismiss the sync at android studio

**Comment by mosabab** [comment]
@breautek @timbru31 
I revert the kotlin as previous to 1.3.50

And keep the gradle version to 6.6.1
And google plugin gradle version to 4.0.1

Could you please review it now

Thanks

**Review by timbru31 — APPROVED** [review]
Changes look good to me, the changelogs for Gradle 6.6 mention nothing special. I'd like a second look, though.

**Comment by breautek** [comment]
I'll review later tonight after work.

**Review by breautek — APPROVED** [review]

**Comment by breautek** [comment]
Looks good to me, merging.

**Pull request merged**